app/javascript/components/StatisticsCollection.js

```diff
@@ -2,7 +2,13 @@ import React from "react";
 import { Query } from "react-apollo";
 import Statistic from "./Statistic";
 
-const StatisticsGroup = (statistics, showHeader, title) => {
+const StatisticsGroup = ({
+  statistics,
+  showHeader = true,
+  title,
+  showRemove = false,
+  weekInReviewId
+}) => {
```

Comment:
could try `defaultProps` rather than default args, a more React way to handle this?

Reply:
I know you can define `defaultProps` inside of a class component, but what about a functional/stateless component?  It looks like I can add this _outside_ of the `const` definition:

```javascript

StatisticsGroup.defaultProps = {
  showHeader: true,
  showRemove: false
};
```

Is there any way to define them _inside_ of the component?  If no, which way is the best practice?